### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by Benawi**
### General requirements
- [x] Make sure that there are no linter errors.
- [x] Make sure that you used the correct Gitflow.
- [x] Make sure that you documented your work in a professional way.
### JavaScript requirements
- [x] Follow our list of best practices for JavaScript.
### Project requirements
- [x] The easiest way to create a React application is using Create React App (CRA) and following the instructions in this [guide](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
- [x] The application should run in the browser without errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```
Comment (divinecharlotte) [inline package.json]: - [ ] kindly consider fixing the error arising in this file to make your app run without errors either in your terminal or the browser as well check the image below to understand clearly the error am talking about there is also a hint to fix it 
Screen Shot 2023-06-05 at 21 32 59

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Benawi        , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 
- Descriptive readme 👍
- Linters are passing 👍  
- Descriptive pr 👍 

 




 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benawi** [comment]
## Changes ♻
- [x]  Page loadign error fixed [comment](https://github.com/Benawi/Microverse-Reac-Math-Magicians/pull/1#discussion_r1218509872)

image
image

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @Benawi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**